app/models/manageiq/providers/azure/cloud_manager/vm_or_template_shared/scanning.rb

```diff
@@ -1,4 +1,16 @@
 module ManageIQ::Providers::Azure::CloudManager::VmOrTemplateShared::Scanning
+  extend ActiveSupport::Concern
```

Comment:
I think `VmOrTemplateShared` also needs to `extend ActiveSupport::Concern`
Is this also in this PR?